doc/classes/GPUParticles3D.xml

```diff
@@ -127,6 +127,7 @@
 		</member>
 		<member name="preprocess" type="float" setter="set_pre_process_time" getter="get_pre_process_time" default="0.0">
 			Amount of time to preprocess the particles before animation starts. Lets you start the animation some time after particles have started emitting.
+			[b]Note:[/b] This can be very expensive if set to a high number as it requires running the particle shader a number of times equal to the [member fixed_fps] (or 30, if [member fixed_fps] is 0) for every second. In extreme cases it can even lead to a GPU crash due to the volume of work done in a single frame.
```

Comment:
```suggestion
			[b]Note:[/b] This can be very expensive if set to a high number as it requires running the particle shader a number of times equal to the [member fixed_fps] (or 30, if [member fixed_fps] is 0) for every second. In extreme cases, it can even lead to a GPU crash due to the volume of work done in a single frame.
```